src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -4248,4 +4280,29 @@ public function get_doctype_info(): ?WP_HTML_Doctype_Info {
 	 * @since 6.5.0
 	 */
 	const COMMENT_AS_INVALID_HTML = 'COMMENT_AS_INVALID_HTML';
+
+	/**
+	 * No-quirks mode document compatability mode.
+	 *
+	 * > In no-quirks mode, the behavior is (hopefully) the desired behavior
+	 * > described by the modern HTML and CSS specifications.
```

Comment:
whence comes this quote?

Reply:
even though the Tag Processor doesn't parse HTML differently because of these, it would be nice to still discuss the impact of TABLE elements closing an open P. these constants will still provide hover-documentation inside the HTML Processor code, and could inform someone, who is poking around and reading the docs, that this has such an impact.

Reply:
> whence comes this quote? and what does "hopefully" communicate to someone trying to figure out the purpose of this mode?

This PR moves this, it was introduced in Processor State in https://github.com/WordPress/wordpress-develop/pull/6972 / [[58779]](https://core.trac.wordpress.org/changeset/58779). I'm not sure where it came from originally.

> even though the Tag Processor doesn't parse HTML differently because of these, it would be nice to still discuss the impact of TABLE elements closing an open P. these constants will still provide hover-documentation inside the HTML Processor code, and could inform someone, who is poking around and reading the docs, that this has such an impact.

I struggled with how to document different behavior without being confusing or too length. I'll add some notes again and see what you think.

Reply:
See bb6d772ff188fc5b765a9dbf2808a89bfd2503c6 for notes about quirks-mode.

Reply:
Found it; this is why I need review and scrutiny on my own contributions. 🙃

https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode